src/sentry/auth/access.py

```diff
@@ -1044,10 +1053,15 @@ def from_request(
             org_member=(summarize_member(member) if member is not None else None),
         ).sso_state
 
+        superuser_scopes = SUPERUSER_SCOPES
+        if not is_self_hosted() and features.has("auth:superuser-read-write", member):
+            if "superuser.write" not in sso_state.permissions:
```

Comment:
nit: same comment

Reply:
if we need this check in multiple places, could we make a util/helper func for this that returns the proper scope?

Reply:
good point!